### ISSUE TITLE
Windows Plugin is not working

### ISSUE THREAD

**Issue by cserb** [Tehnix/spaceneovim#13]
The windows plugin is not working as expected. The Readme states I should be able to move via `SPC 0` (`SPC [number]`) but it doesn't seem to work

```
let g:dotspaceneovim_configuration_layers = [
  \  '+nav/buffers'
  \, '+nav/files'
  \, '+nav/quit'
  \, '+nav/windows'
  \, '+nav/start-screen'
  \, '+nav/text'
  \, '+checkers/neomake'
  \, '+completion/deoplete'
  \, '+tools/terminal'
  \, '+ui/airline'
  \,'+ui/toggles'
  \]
```

Also I can't  figure out where to add plugins via vim-plug

**Comment by Vm900** [comment]
On the vim-plug issue,you can always use your normal .vimrc.
Hope that helps

**Comment by Tehnix** [comment]
Yeah, currently the window commands are not working, I'm looking into that as per [https://github.com/Tehnix/spaceneovim-layers/issues/1](https://github.com/Tehnix/spaceneovim-layers/issues/1)

**Comment by Tehnix** [comment]
This has now been resolved, see Tehnix/spaceneovim-layers#1 for more info.
